Mathlib/Algebra/MonoidAlgebra/Basic.lean

```diff
@@ -314,6 +314,12 @@ theorem domCongr_refl : domCongr k A (.refl G) = .refl :=
 @[to_additive (attr := simp)]
 theorem domCongr_symm (e : G ≃* H) : (domCongr k A e).symm = domCongr k A e.symm := rfl
 
+@[to_additive (attr := simp)]
+theorem trans_domCongr_domCongr {I : Type*} [Monoid I] (e : G ≃* H) (f : H ≃* I) :
+    (domCongr k A e).trans (domCongr k A f) = domCongr k A (e.trans f) := by
```

Comment:
Is this really the right direction for this lemma? 

the non-equiv variant `mapDomainAlgHom_comp` (which is also a `simp` lemma) is in the opposite direction, this suggests that this should be
```suggestion
    domCongr k A (e.trans f) = (domCongr k A e).trans (domCongr k A f) := by
```

(which would rename the lemma `domCongr_trans`)